src/home/App.js

```diff
@@ -52,7 +52,7 @@ class App extends Component {
             defaultMessage="Data Management"
           />
         ),
-        bgImage: templateIcon
+        bgImage: templateTileImage
```

Comment:
I find this name is quite misleading, it should be something like dataManagementTileImage. You also have a templatesTileImage in Alarms.js

Reply:
Thank you! :) Forgot to rename it.

Reply:
You made my day. :)